### PR TITLE
feat(perf-views) Move performance charts to the sidebar

### PR DESCRIPTION
**Pull request by markstory**
The apdex, error rate and throughput charts are most useful when they can be viewed alongside duration breakdowns and other charts. Adding stripped down charts to the sidebar enables these comparisons.

Users can also use sidebar charts to zoom in on problems reloading the entire page.

Screen Shot 2020-05-07 at 2 44 06 PM

### CODE REVIEW

**Review by wmak — APPROVED** [review]
👏 Dang this is very nice.
Nit? Request? could we have the line on all 4 charts instead of either just the 3 on the side or on the top chart?

**Comment by markstory** [comment]
>  Nit? Request? could we have the line on all 4 charts instead of either just the 3 on the side or on the top chart?

Do you mean the hover state line? Having it on all graphs is hard as axis can only be shared across charts in the same instance. That is tricky here as the charts are split into different parts of the layout.

**Comment by wmak** [comment]
Yeah the hover state line, I don't think its the end of the world without it, but would make it much easier to compare across all 4 graphs.

File: src/sentry/static/sentry/app/views/performance/transactionSummary/sidebarCharts.tsx [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+import {StyledIconQuestion} from '../styles';
+
+type Props = ReactRouter.WithRouterProps & {
+  api: Client;
+  organization: OrganizationSummary;
+  location: Location;
+  eventView: EventView;
+};
+
+type State = {
```
Comment (dashed) [inline src/sentry/static/sentry/app/views/performance/transactionSummary/sidebarCharts.tsx:41]: unused

**Review by dashed — APPROVED** [review]

**Pull request merged**